static/app/bootstrap/initializeSdk.tsx

```diff
@@ -169,12 +169,10 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
     },
   });
 
-  if (process.env.NODE_ENV !== 'production') {
-    if (sentryConfig.environment === 'development') {
```

Comment:
With `dev-ui` this was `prod` so spotlight was not getting loaded

Reply:
i dont have context on the original PR, but one thing to note is most bundlers look for this process.env.NODE_ENV check to treeshake out dev code

Reply:
`process.env.NODE_ENV` is still there, it's only the runtime check that is removed

Reply:
@billyvg did you check the produced prod bundle?
I am not sure if I tested exactly what you have now, but this setup made sure Spotlight is not contained in a production build.
If this is still the case, then LGTM

Reply:
@HazAT Yeah I checked with webpack-bundle-analyzer and it's not there for prod builds